flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskExecutor.java

```diff
@@ -138,12 +140,8 @@
 import org.apache.flink.util.concurrent.ExecutorThreadFactory;
 import org.apache.flink.util.concurrent.FutureUtils;
 
-import org.apache.flink.shaded.guava30.com.google.common.collect.ImmutableList;
-import org.apache.flink.shaded.guava30.com.google.common.collect.Sets;
-
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
-
 import java.io.File;
```

Comment:
Checkstyle failed because of imports of order.
FYI: [build](https://dev.azure.com/khachatryanroman/flink/_build/results?buildId=1590&view=results) after fixing imports.

Reply:
Got it, thanks.